amt/api/routes/algorithm.py

```diff
@@ -215,6 +216,7 @@ async def get_algorithm_context(
 
 
 @router.get("/{algorithm_id}/details")
+@add_permissions(permissions={AuthorizationResource.ALGORITHM: [AuthorizationVerb.READ]})
```

Comment:
Could we maybe simplify this (and all other instances) a little to this? 
```suggestion
@permission({ALGORITHM: [READ]})
```

Reply:
I renamed the decorated and removed the permissions key. I kept the enums, solutions for using key only without the Enum are not elegant and seem more work and more error prone.